trie/trie.go

```diff
@@ -116,16 +119,20 @@ func (t *Trie) Put(key, value []byte) error {
 func (t *Trie) tryPut(key, value []byte) (err error) {
 	k := keyToNibbles(key)
 	var n node
+	var ok bool
 
 	if len(value) > 0 {
-		_, n, err = t.insert(t.root, k, &leaf{key: nil, value: value, dirty: true})
+		ok, n, err = t.insert(t.root, k, &leaf{key: nil, value: value, dirty: true})
 	} else {
-		_, n, err = t.delete(t.root, k)
+		ok, n, err = t.delete(t.root, k)
 	}
 
 	if err != nil {
 		return err
 	}
+	if !ok {
+		log.Warn("tryPut returned not ok for operation insert/delete")
```

Comment:
I don't think we need to be logging this, it'll probably clog up the logs too much. if it's !ok, it's not an issue, if there was an issue there would be an error

Reply:
well, the issue is that the ok is never used anywhere and lint complained about it. So I made use of it by adding a log. The other option is to delete the instruction all together from the method and stop returning the ok since its not used for anything. What is the preferred option ?

Reply:
let's just remove `ok`, if it's not ok, then there is an error returned so it is redundant

Reply:
yep, oks

Reply:
I've raised https://github.com/ChainSafe/gossamer/issues/617 in order to have this fixed, since after few changes the risk and the nature of the fix will add extra logic that is not related to the ticket, possibly creating a scope creep here.
If you are ok I would like to merge like this and remove the ok in another ticket.

Reply:
sure, are you planning to work on #617 ?

Reply:
yes